dev/com.ibm.ws.request.timing_fat/fat/src/com/ibm/ws/request/timing/fat/HungRequestTiming.java

```diff
@@ -75,10 +74,6 @@ public void setupTestStart() throws Exception {
         if (server != null && !server.isStarted()) {
             server.startServer();
         }
-        // Need to ensure the configuration is finished updating before starting a test
-        server.setServerConfigurationFile("server_original.xml");
-        server.waitForStringInLog("CWWKG0017I", 90000);
-        server.setMarkToEndOfLog();
```

Comment:
I thought we had added this here, to ensure every test starts of with a fresh server config, since we saw some test failures in the past, where the server config wasn't updated properly for subsequent tests?